src/Util.ts

```diff
@@ -26,16 +22,20 @@ export const EMPTY_FUNCTION = () => {};
  * Efficient and high resolution timestamp in milliseconds elapsed since {@link Util.timeOrigin}
  */
 export function time(): number {
-    return Math.floor(_perf.now());
+    return Math.floor(performance.now());
 }
 
 /**
  * Time origin represents the time when the application has started
  */
 export function timeOrigin(): number {
```

Comment:
Why not
export function timeOrigin(): number {
    return Math.floor(_perf.timeOrigin);
}
See => https://developer.mozilla.org/en-US/docs/Web/API/Performance/timeOrigin

With your proposition the definition of timeOrigin change, normally it must check =>

`unixTime= performance.timeOrigin + performance.now();`